tests/Database/Consensus/ConsensusTest.nim

```diff
@@ -0,0 +1,254 @@
+#Consensus DB Test.
+
+#Test lib.
+import unittest2
+
+#Util lib.
+import ../../../src/lib/Util
+
+#Hash lib.
+import ../../../src/lib/Hash
+
+#MinerWallet lib.
+import ../../../src/Wallet/MinerWallet
+
+#Merit lib.
+import ../../../src/Database/Merit/Merit
+
+#Consensus lib.
+import ../../../src/Database/Consensus/Consensus
+
+#Transactions lib.
+import ../../../src/Database/Transactions/Transactions
+
+#Test Database lib.
+import ../TestDatabase
+
+#Test Merit lib.
+import ../Merit/TestMerit
+
+#Compare Consensus lib.
+import CompareConsensus
+
+#Random standard lib.
+import random
+
+#Tables lib.
+import tables
+
+suite "Consensus":
+    setup:
+        #Seed random.
+        randomize(int64(getTime()))
+
+    test "Verify.":
```

Comment:
Can we get a more descriptive test label please?

Reply:
The only assertion I can see in this test is the comparing of the consensus against a reloaded consensus. Renaming it after that unless you have a better name.